teuthology/exceptions.py

```diff
@@ -50,21 +54,30 @@ class CommandFailedError(Exception):
     """
     Exception thrown on command failure
     """
-    def __init__(self, command, exitstatus, node=None, label=None):
-        self.command = command
+    def __init__(self, cmd, exitstatus, node=None, label=None):
+        self.cmd = cmd
```

Comment:
Why renaming `command` to `cmd`? 
- This change is not necessary.
- Breaks the convention of naming arguments.

Reply:
> Why renaming command to cmd?

Lesser effort to type, and easier to keep statements under 79 characters which in turn saves us from hard to read split-over-several-line statements .